mmv1/third_party/terraform/services/appengine/resource_app_engine_flexible_app_version_test.go

```diff
@@ -166,18 +187,38 @@ resource "google_project" "my_project" {
   billing_account = "%{billing_account}"
 }
 
-resource "google_app_engine_application" "app" {
-  project     = google_project.my_project.project_id
-  location_id = "us-central"
-}
-
 resource "google_project_service" "project" {
+  for_each = toset([
```

Comment:
```
        Error: Missing resource instance key
        
          on terraform_plugin_test.tf line 22, in resource "google_compute_network" "network":
          22:   project                 = google_project_service.project.project
        
        Because google_project_service.project has "for_each" set, its attributes
        must be accessed on specific instances.
        
        For example, to correlate with indices of a referring resource, use:
            google_project_service.project[each.key]
```

Let's just add separate blocks instead of using for_each here!

Reply:
Pfiou I missed that. Done